internal/execute/outputs.go

```diff
@@ -260,13 +261,165 @@ func generateOptionOutput(
 	// !!! const colors = createColors(sys);
 
 	// name and description
-	// !!! name := getDisplayNameTextOfOption(option)
+	name := getDisplayNameTextOfOption(option)
+
+	// value type and possible value
+	valueCandidates := getValueCandidate(option)
+
+	var defaultValueDescription string
+	if msg, ok := option.DefaultValueDescription.(*diagnostics.Message); ok {
+		defaultValueDescription = msg.Format()
+	} else {
+		defaultValueDescription = formatDefaultValue(
+			option.DefaultValueDescription,
+			core.IfElse(
+				option.Kind == tsoptions.CommandLineOptionTypeList || option.Kind == tsoptions.CommandLineOptionTypeListOrElement,
+				option.Elements(), option,
+			),
+		)
+	}
 
-	// !!!
+	var terminalWidth int
+	// const terminalWidth = sys.getWidthOfTerminal?.() ?? 0;
+
+	// Note: child_process might return `terminalWidth` as undefined.
+	if terminalWidth >= 80 {
+		// !!!     let description = "";
+		// !!!     if (option.description) {
+		// !!!         description = getDiagnosticText(option.description);
+		// !!!     }
+		// !!!     text.push(...getPrettyOutput(name, description, rightAlignOfLeft, leftAlignOfRight, terminalWidth, /*colorLeft*/ true), sys.newLine);
+		// !!!     if (showAdditionalInfoOutput(valueCandidates, option)) {
+		// !!!         if (valueCandidates) {
+		// !!!             text.push(...getPrettyOutput(valueCandidates.valueType, valueCandidates.possibleValues, rightAlignOfLeft, leftAlignOfRight, terminalWidth, /*colorLeft*/ false), sys.newLine);
+		// !!!         }
+		// !!!         if (defaultValueDescription) {
+		// !!!             text.push(...getPrettyOutput(getDiagnosticText(Diagnostics.default_Colon), defaultValueDescription, rightAlignOfLeft, leftAlignOfRight, terminalWidth, /*colorLeft*/ false), sys.newLine);
+		// !!!         }
+		// !!!     }
+		// !!!     text.push(sys.newLine);
+	} else {
```

Comment:
It would be nice to leave the `// !!! ` comment near the `terminalWidth` declaration, since that's the part/reason that this is not yet implemented